styles/styles.css

```diff
@@ -1263,19 +1264,25 @@ main.blue-contract .section.section-with-title p {
 
 .redesign-v2 .pretitle {
   font-feature-settings: 'clig' off, 'liga' off;
-  font-size: var(--f-subtitle-2-font-size);
-  letter-spacing: var(--f-subtitle-2-letter-spacing);
-  line-height: var(--f-subtitle-2-line-height);
+  font-size: var(--f-body-2-font-size);
```

Comment:
This change is going to break down: https://github.com/Netcentric/vg-volvotrucks-us-rd/pull/83
I agree that all pretitles should look the same, but which one is the one that we should follow?